RecoTauTag/HLTProducers/src/CaloTowerFromL1TCreatorForTauHLT.cc

```diff
@@ -0,0 +1,107 @@
+          bool isAccepted = false;
+          if (mVerbose == 2) {
+              edm::LogInfo("JetDebugInfo") << "CaloTowerFromL1TCreatorForTauHLT::produce-> " << idx 
+                                           << " tower et/eta/phi/e: "                        << cal->et()  << '/' 
+                                                                                             << cal->eta() << '/' 
+                                                                                             << cal->phi() << '/' 
+                                                                                             << cal->energy() 
+                                           << " is...";
+              }
+          if (cal->et() >= mEtThreshold && cal->energy() >= mEThreshold ) {
+            math::PtEtaPhiELorentzVector p( cal->et(), cal->eta(), cal->phi(), cal->energy() );
+            double delta  = ROOT::Math::VectorUtil::DeltaR((*myL1Jet).p4().Vect(), p);
+            if(delta < mCone) {
+              isAccepted = true;
+              Sum08 += cal->et(); 
+              cands->push_back( *cal );
+            }
+          }
+          if (mVerbose == 2){
+              if (isAccepted) edm::LogInfo("JetDebugInfo") << "accepted \n";
+              else edm::LogInfo("JetDebugInfo") << "rejected \n";
+            }
+        }
+      }
+      idTau++;
+    }
+  } 
+  else {
+    edm::LogWarning("MissingProduct") << "L1Upgrade jet bx collection not found." << std::endl;
+  }
+  
+  evt.put( cands );
+  
+}
+
+void CaloTowerFromL1TCreatorForTauHLT::fillDescriptions( edm::ConfigurationDescriptions & desc ) {
+
+  edm::ParameterSetDescription aDesc;
+
+  aDesc.add<edm::InputTag>("TauTrigger"     , edm::InputTag("caloStage2Digis"))->setComment("L1 Tau collection for seeding"                                );
+  aDesc.add<edm::InputTag>("towers"         , edm::InputTag("towerMaker"     ))->setComment("Input tower collection"                                       );
+  aDesc.add<int>          ("TauId"          , 0                               )->setComment("Item from L1 Tau collection used for seeding. From 0 to 11"   );
+  aDesc.add<double>       ("UseTowersInCone", 0.8                             )->setComment("Radius of cone around seed"                                   );
+  aDesc.add<double>       ("minimumE"       , 0.8                             )->setComment("Minimum tower energy"                                         );
+  aDesc.add<double>       ("minimumEt"      , 0.5                             )->setComment("Minimum tower ET"                                             );
+  aDesc.addUntracked<int> ("verbose"        , 0                               )->setComment("Verbosity level; 0=silent"                                    );
+  aDesc.addUntracked<int> ("BX"             , 0                               )->setComment("Set bunch crossing; 0 = in time, -1 = previous, 1 = following");
+
+  desc.add                ("caloTowerMakerHLT", aDesc);
```

Comment:
The string should be the calss name starting with lower-case hlt.
Also to be different from the one in the old module!